sequencer.c

```diff
@@ -2524,7 +2524,7 @@ static int create_seq_dir(void)
 {
 	if (file_exists(git_path_seq_dir())) {
 		error(_("a cherry-pick or revert is already in progress"));
-		advise(_("try \"git cherry-pick (--continue | --quit | --abort)\""));
+		advise(_("try \"git cherry-pick (--continue | --skip | --quit | --abort)\""));
```

Comment:
Didn't your tests in the first commit check that 'cherry-pick --skip' would fail if you were in the middle of a revert and require a 'revert --skip'?  If so, does this advice need to be corrected?

Reply:
Yes, using `cherry-pick --skip` in case of `revert` should not work. Also `--continue` flag is a no-go. We can tackle it by having a new commit changing the signature of `create_seq_dir` to `static int create_seq_dir(struct repository *)` and then conditionally pinpointing and providing advice?

Because simply changing to: `try git cherry-pick (--abort | --quit)` or `try git (cherry-pick | revert) (--continue | --skip | --quit | --abort)` does no proper justice to commands.

Reply:
@newren @tgummerer I really need a second opinion on this one.

Reply:
Looking at how `--continue` works, I wonder if the right thing to do *for now* is to not make `cherry-pick --skip` fail when a revert is in progress and vice versa.

I feel like longer term we might want to disallow `cherry-pick --{continue,skip,...}` when a revert is in progress and the other way around, however that's not something this patch (or your project should be concerned with imho).

Just continuing with this UI weirdness might be the simplest solution for now, and doesn't make `--skip` special.

In general anything that makes `--skip` different than how `--continue` and friends work might not be great, as it might be harder to unify how they work in the future, and makes it somewhat hard to explain why one is different than the others.

Reply:
You could just do the really simple:
		advise(_("try \"git cherry-pick (--continue | --skip | --quit | --abort)\""));
		advise(_("or \"git revert (--continue | --skip | --quit | --abort)\""));
Yeah, it's not as tight as it could be but it seems we didn't care before ("a cherry-pick or revert is already in progress" -- rather than finding out which operation it was and telling the user).